src/Microsoft.VisualStudio.AppDesigner/Common/Utils.vb

```diff
@@ -705,5 +707,28 @@ Namespace Microsoft.VisualStudio.Editors.AppDesCommon
         End Class
 #End Region
 
+        Public Class SerializationProvider
+
+            Public Shared Sub Serialize(stream As Stream, value As Object)
+                If value Is Nothing Then
+                    Return
```

Comment:
In this case no data is written to the stream. Does a deserialize call on the same data return `null` correctly?

Reply:
I've added a unit test for this and updated these guard clauses to be more encompassing.